### PR TITLE
Add Remix docset

### PR DESCRIPTION
**Pull request by gazoakley**
Hi - this is a first contributed docset for me, so feedback/comments appreciated

### CODE REVIEW

File: Remix/docset.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Remix",
+    "version": "1.9.0",
+    "archive": "remix.tgz",
```
Comment (Kapeli) [inline Remix/docset.json:4]: This file doesn't exist. You added `Remix.tgz` instead (note: case-sensitive).

**Comment by Kapeli** [comment]
The white icon isn't really a good idea. It won't look ok in Dash.

File: Remix/docset.json (hunk 1 above)
Comment (gazoakley) [reply Remix/docset.json]: Fixed - and the docset isn't in the `docsets` directory either 👍

**Comment by Kapeli** [comment]
Thank you! ❤️

**Pull request merged**